scss/fonts.scss

```diff
@@ -0,0 +1,4 @@
+@font-face {
+  font-family: "PT MON";
```

Comment:
It's the same font. Css nodules use different format for importing fonts. You can't just `@import "../public/PTMono-Regular.ttf";`

Reply:
Please fix typo.

Reply:
I think this font-family is incorrect, should actually be this:

```suggestion
  font-family: "PT Mono";
```

Reply:
Fixed.